### ISSUE TITLE
Warning: WebView has been extracted from react-native core, React Native 0.58+

### ISSUE THREAD

**Issue by Jaraujo6** [software-mansion/react-native-gesture-handler#456]
As of React Native 0.58, WebView has been decoupled and is now published from https://github.com/react-native-community/react-native-webview

As a result, every time your app runs you will see a warning message with the above message. Please update the GestureHandlers.js to import WebView from the new npm module instead of react-native module.

**Comment by luizbaldi** [comment]
Same issue here, for now I'm just ignoring this error message with the snippet below:

```javascript
import { YellowBox } from 'react-native'

YellowBox.ignoreWarnings(['WebView'])
```

I'm also available to open a PR with this fix if necessary :)

**Comment by joge17** [comment]
When will this be fixed?

**Comment by jakub-gonet** [comment]
Should be resolved by #438
